tests/modules/individuals/resources/test_elasticsearch.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+
+from tests import utils as test_utils
+from tests.modules.elasticsearch.resources import utils as es_utils
+from tests.modules.individuals.resources import utils as individual_utils
+
+
+# OK I don't understand why this does not guarantee that the mapping is generated JP??
```

Comment:
@bluemellophone Any help on why the ES mapping returns {} would be much appreciated.

Reply:
Following up on this, the reason the mappings are empty is that the testing code doesn't use the same index names as the production code.  The code was looking at index name `/api/v1/search/testing.app.modules.individuals.models.individual/mappings` but should be using `/api/v1/search/app.modules.individuals.models.individual/mappings`.

Reply:
I'm pushing a new PR soon to address this issue

Reply:
See here: https://github.com/WildMeOrg/houston/pull/586